axelor-stock/src/main/resources/views/StockMoveLine.xml

```diff
@@ -229,7 +229,7 @@
 	</grid>
 
     <form name="stock-move-line-all-form" title="Stock move line" model="com.axelor.apps.stock.db.StockMoveLine"
-    	onNew="action-stock-move-line-group-onnew" onLoad="action-stock-move-line-group-all-onload" readonlyIf="stockMove.statusSelect == 3" width="large"
+    	onNew="action-stock-move-line-group-onnew" onLoad="action-stock-move-line-group-all-onload" readonlyIf="stockMove.statusSelect > 3" width="large"
```

Comment:
remove readonlyIf condition and put it per panel according to status, only those fields which are mentioned on ticket  should be editable not all